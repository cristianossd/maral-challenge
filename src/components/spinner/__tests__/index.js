import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import Spinner from '../';

describe('<Spinner />', () => {
  it('render correctly', () => {
    const wrapper = shallow(<Spinner />);

    expect(wrapper.find('.Spinner').length).toBe(1);
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
