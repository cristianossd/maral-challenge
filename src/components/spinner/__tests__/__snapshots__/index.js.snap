// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Spinner /> render correctly 1`] = `
<svg
  className="Spinner"
  preserveAspectRatio="xMidYMid"
  viewBox="0 0 100 100"
>
  <g
    transform="rotate(0 50 50)"
  >
    <rect
      fill="#337ab7"
      height="12"
      rx="9.4"
      ry="4.8"
      width="6"
      x="47"
      y="24"
    >
      <animate
        attributeName="opacity"
        begin="-0.9166666666666666s"
        dur="1s"
        repeatCount="indefinite"
        times="0;1"
        values="1;0"
      />
    </rect>
  </g>
  <g
    transform="rotate(30 50 50)"
  >
    <rect
      fill="#337ab7"
      height="12"
      rx="9.4"
      ry="4.8"
      width="6"
      x="47"
      y="24"
    >
      <animate
        attributeName="opacity"
        begin="-0.8333333333333334s"
        dur="1s"
        repeatCount="indefinite"
        times="0;1"
        values="1;0"
      />
    </rect>
  </g>
  <g
    transform="rotate(60 50 50)"
  >
    <rect
      fill="#337ab7"
      height="12"
      rx="9.4"
      ry="4.8"
      width="6"
      x="47"
      y="24"
    >
      <animate
        attributeName="opacity"
        begin="-0.75s"
        dur="1s"
        repeatCount="indefinite"
        times="0;1"
        values="1;0"
      />
    </rect>
  </g>
  <g
    transform="rotate(90 50 50)"
  >
    <rect
      fill="#337ab7"
      height="12"
      rx="9.4"
      ry="4.8"
      width="6"
      x="47"
      y="24"
    >
      <animate
        attributeName="opacity"
        begin="-0.6666666666666666s"
        dur="1s"
        repeatCount="indefinite"
        times="0;1"
        values="1;0"
      />
    </rect>
  </g>
  <g
    transform="rotate(120 50 50)"
  >
    <rect
      fill="#337ab7"
      height="12"
      rx="9.4"
      ry="4.8"
      width="6"
      x="47"
      y="24"
    >
      <animate
        attributeName="opacity"
        begin="-0.5833333333333334s"
        dur="1s"
        repeatCount="indefinite"
        times="0;1"
        values="1;0"
      />
    </rect>
  </g>
  <g
    transform="rotate(150 50 50)"
  >
    <rect
      fill="#337ab7"
      height="12"
      rx="9.4"
      ry="4.8"
      width="6"
      x="47"
      y="24"
    >
      <animate
        attributeName="opacity"
        begin="-0.5s"
        dur="1s"
        repeatCount="indefinite"
        times="0;1"
        values="1;0"
      />
    </rect>
  </g>
  <g
    transform="rotate(180 50 50)"
  >
    <rect
      fill="#337ab7"
      height="12"
      rx="9.4"
      ry="4.8"
      width="6"
      x="47"
      y="24"
    >
      <animate
        attributeName="opacity"
        begin="-0.4166666666666667s"
        dur="1s"
        repeatCount="indefinite"
        times="0;1"
        values="1;0"
      />
    </rect>
  </g>
  <g
    transform="rotate(210 50 50)"
  >
    <rect
      fill="#337ab7"
      height="12"
      rx="9.4"
      ry="4.8"
      width="6"
      x="47"
      y="24"
    >
      <animate
        attributeName="opacity"
        begin="-0.3333333333333333s"
        dur="1s"
        repeatCount="indefinite"
        times="0;1"
        values="1;0"
      />
    </rect>
  </g>
  <g
    transform="rotate(240 50 50)"
  >
    <rect
      fill="#337ab7"
      height="12"
      rx="9.4"
      ry="4.8"
      width="6"
      x="47"
      y="24"
    >
      <animate
        attributeName="opacity"
        begin="-0.25s"
        dur="1s"
        repeatCount="indefinite"
        times="0;1"
        values="1;0"
      />
    </rect>
  </g>
  <g
    transform="rotate(270 50 50)"
  >
    <rect
      fill="#337ab7"
      height="12"
      rx="9.4"
      ry="4.8"
      width="6"
      x="47"
      y="24"
    >
      <animate
        attributeName="opacity"
        begin="-0.16666666666666666s"
        dur="1s"
        repeatCount="indefinite"
        times="0;1"
        values="1;0"
      />
    </rect>
  </g>
  <g
    transform="rotate(300 50 50)"
  >
    <rect
      fill="#337ab7"
      height="12"
      rx="9.4"
      ry="4.8"
      width="6"
      x="47"
      y="24"
    >
      <animate
        attributeName="opacity"
        begin="-0.08333333333333333s"
        dur="1s"
        repeatCount="indefinite"
        times="0;1"
        values="1;0"
      />
    </rect>
  </g>
  <g
    transform="rotate(330 50 50)"
  >
    <rect
      fill="#337ab7"
      height="12"
      rx="9.4"
      ry="4.8"
      width="6"
      x="47"
      y="24"
    >
      <animate
        attributeName="opacity"
        begin="0s"
        dur="1s"
        repeatCount="indefinite"
        times="0;1"
        values="1;0"
      />
    </rect>
  </g>
</svg>
`;
